src/docregistry/plugin.ts

```diff
@@ -21,6 +21,16 @@ const docRegistryProvider: JupyterLabPlugin<IDocumentRegistry> = {
     let registry = new DocumentRegistry();
     registry.addModelFactory(new TextModelFactory());
     registry.addModelFactory(new Base64ModelFactory());
+    registry.addFileType({
+      name: 'Text',
+      extension: '.txt',
+      fileType: 'file',
+      fileFormat: 'text'
+    });
+    registry.addCreator({
+      name: 'Text File',
+      fileType: 'Text',
```

Comment:
I think we should change the one in services to `ContentType` (if you agree I'll open an issue there because I am in the middle of a major refactor there, and we can merge this for now).